Converter/EntityConverter.php

```diff
@@ -97,10 +98,7 @@ public function createNewObject($class, Request $request, $create, $options)
         // Convert array into object and test validity
         $object = $this->crudTransformer->arrayToObject($data, $class);
         if ($object instanceof ValidatableInterface && $create) {
-            $violations = $this->crudTransformer->arrayToObjectPropertyValidation($data, $class);
-            $valid = !((bool) $violations->count());
-            $object->setViolations($violations);
-            $object->setValid($valid);
+            $this->validateNewObject($object, $class, $data, $options);
```

Comment:
It looks like unit-test for EntityConverter::createNewObject needs to be updated.